mappings/net/minecraft/client/render/FirstPersonRenderer.mapping

```diff
@@ -31,6 +31,8 @@ CLASS dkp net/minecraft/client/render/FirstPersonRenderer
 		ARG 3 item
 	METHOD a resetEquipProgress (Lahf;)V
 	METHOD a renderArm (Lait;)V
+	METHOD a (Lait;F)V
```

Comment:
Just noting that I have not studied this method specifically, and have therefore not mapped it fully. I merely mapped the relating parameter.